junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -349,6 +423,7 @@ private static Optional<IndicativeSentencesGeneration> findIndicativeSentencesGe
 				SearchOption.INCLUDE_ENCLOSING_CLASSES);
 		}
 
+		@SuppressWarnings("SameParameterValue")
```

Comment:
Is this specific to IntelliJ IDEA?

If so, we don't need this.

Reply:
It is and I find it helpful to ignore these because it makes it easier to spot new/other warnings.

Reply:
```suggestion
```